extensions/ql-vscode/test/pure-tests/remote-queries/markdown-generation/markdown-generation.test.ts

```diff
@@ -3,6 +3,8 @@ import * as path from 'path';
 import * as fs from 'fs-extra';
 import { generateMarkdown } from '../../../../src/remote-queries/remote-queries-markdown-generation';
 
+const expectedFileNames = ['summary', 'github-codeql', 'meteor-meteor'];
```

Comment:
This introduces knowledge about the test data outside of the test data dir and into the source code. It works, but it's not ideal IMO.

Instead of keeping the list of expected file names and then looping over them etc. we could move all the files we expect into a subdir e.g. `expected` and then look at all the contents of the dir and iterating over those.

Reply:
Done in latest commits! Thanks for helping with this 🍐 🎉